_docs/tasks/traffic-management/egress-tcp.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Control Egress TCP Traffic
+overview: Describes how to configure Istio to route TCP traffic from services in the mesh to external services.
+
+order: 41
+
+layout: docs
+type: markdown
+---
+{% include home.html %}
+
+The [Control Egress Traffic]({{home}}/docs/tasks/traffic-management/egress.html) task demonstrated how external (out of the mesh) HTTP and HTTPS services can be accessed from applications inside the mesh. To remind: by default, Istio-enabled applications are unable to access URLs outside of the mesh. To enable such access, [egress rules]({{home}}/docs/reference/config/istio.routing.v1alpha1.html#EgressRule) must be defined, or, alternatively, [direct access to external services]({{home}}/docs/tasks/traffic-management/egress.html#calling-external-services-directly) must be configured.
```

Comment:
> To remind: by default, Istio-enabled applications are unable to access URLs outside of the mesh.

This is only true when mTLS is enabled, right?

Reply:
Good point.  Actually, pods with sidecars cannot access URLs outside of the Kubernetes cluster. The question is if we should be so specific here.

Reply:
Replaced "outside the mesh" with "outside the cluster".